src/views/Player.vue

```diff
@@ -719,7 +723,8 @@ export default defineComponent({
           title: state.questionSets[index].title,
           paletteItems: states,
           instructionText: paletteInstructionText,
-          maxQuestionsAllowedToAttempt: state.questionSets[index].max_questions_allowed_to_attempt
+          instructionPageText: instructionPageText,
```

Comment:
maybe name the column variable differently? 
`instructionPageText` : `localInstructionPageText`